hebi_a-2085-05-parallel-gripper_moveit_config/config/A-2085-05.srdf

```diff
@@ -3,18 +3,18 @@
     This is a format for representing semantic information about the robot structure.
     A URDF file must exist for this robot as well, where the joints and the links that are referenced are defined
 -->
-<robot name="A-2085-05">
+<robot name="A-2085-05-parallel-gripper">
```

Comment:
Ah -- I see the robot name got changed too.  I think that makes sense; just make sure this is done for the 2084-01, too